Shaders/Materials/BlinnPhong/BlinnPhongMaterial.glsl

```diff
@@ -30,11 +30,18 @@ struct Material
     float ns;
     float alpha;
 
+    int hasPerVertexKd;
+    int renderAsSplat;
```

Comment:
I'm pretty sure this (`m_renderAsSplat`) has nothing to do here and must be managed elsewhere.
About `m_hasPerVertexKd` I'm not sure

Reply:
Why are you so sure ?
In the current state, `m_renderAsSplat` is used in `toDiscard`, and thus needs to be defined in BlinnPhongMaterial.

Reply:
I mean this is not related to the material, only to point clouds.
Thus the material should not have to decide whether or not the fragment should be discarded when splatting, it must be decided on top of it.

Reply:
Yes, but this is only way to do it today.
This should be fixed in v2

Reply:
could you simply use another material ? not sure how to specify it in the loader

Reply:
This minor comment allow to use bling phong as is.
If you prefer and can copy all the shaders for 2 lines of code.

Reply:
I agree that renderAsSplat is not really a part of a BlinnPhong shader. Nevertheless, it is the simplest way to achieve the objective of splat-based rendering. This clearly justify the need for shader composition and on-line glsl code generation to be integrated in v2. For the moment, we can keep it like this (it will be not be the only thing in the vast amount of shaders used by radium to be not really clean). I have an idea to make this clean that relies on shader configuration and will try to implement it as soon as possible on this branch  but this will be a step towards v2 and material-shader polymorphism.
hasPerVertexKd is welcome in BlinnPhong and could be generalized to other materials with a small naming effort. So, we can keep it for now.
To conclude, yes, this is not the good way to implement BlinnPhong for splatting but it works and do not necessitate huge code re-writing based on non validated interfaces. So we can keep it for now.